apps/coscrad-frontend/src/utils/generic-components/presenters/single-property-presenter.tsx

```diff
@@ -9,8 +9,9 @@ export const SinglePropertyPresenter = <T,>({
     value,
     display,
 }: IValueAndDisplay<T>): JSX.Element => (
-    <>
-        <strong>{display}</strong>: {value}
-        <br />
```

Comment:
Seems like we might want a `MultiplePropertyPresenter` that
- handles conditional rendering for optional props
- puts things in a stack (based on your past advice to me)